projects/ad6676-ebz/src/ad6676_ebz.c

```diff
@@ -473,6 +473,9 @@ int main(void)
 	       ad6676_core_param.num_channels, 16);
 
 #ifdef IIO_SUPPORT
+	// Allow time for UART messages to be displayed
+	no_os_mdelay(200);
```

Comment:
we need to fix xilinx uart implementation at some point https://github.com/analogdevicesinc/no-OS/issues/2514